README.md

```diff
@@ -55,10 +55,11 @@ Once you've _VCPKG_, you can build _PCL_ with the next command:
 c:\vcpkg> vcpkg install pcl:x64-windows
 ```
 
-Once finished (be patient) you've to do two last steps:
+Once finished (be patient) you've to do three last steps:
 
 1. Add **C:\vcpkg\installed\x64-windows\bin** to your user's _PATH_.
-2. Set the _CMake Toolchain File_ to **c:\vcpkg\scripts\buildsystems\vcpkg.cmake**.
+2. Add **C:\vcpkg\installed\x64-windows\debug\bin** to your user's _PATH_.
```

Comment:
1. should be added to **system** _PATH_, didn't work for me otherwise
2. install path can different, for me it was `C:\users\roman\vcpkg\installed\x64-windows\debug\bin`

Reply:
1. In my case it works both, with system path and also with user path.
2. In my case I first tried `c:\users\josé ángel\vcpk` but it did't work correctly when installing the packages (maybe because the special characters we have in Spanish or because the space.